tests/suites/test_suite_alignment.function

```diff
@@ -17,6 +17,18 @@ int parse_hex_string(char *hex_string, uint64_t *result)
     if (mbedtls_test_unhexify(raw, sizeof(raw), hex_string, &olen) != 0) {
         return 0;
     }
+
+    /* If < 8 bytes, shift right and pad with leading zeros for big-endian */
+    if (MBEDTLS_IS_BIG_ENDIAN && olen < 8) {
+        int offset = 8 - olen;
```

Comment:
If seems that this whole block could be replaced with a reasonable use of `offset` in the look at (new) line 33